### DISCUSSION TITLE
'brew upgrade' no longer shows recently updated formulas and casks

### DISCUSSION THREAD

**Discussion by nycaleksey** [Homebrew/discussions#456] [Everyday usage] [answered]
Hi!

Homebrew had a nice little feature of showing a list of recently updated formulaes/casks (not necessarily installed) as output of "brew upgrade". Now, after a recent update it only shows the summarized counts like this:

==> Updated Formulae
Updated 66 formulae.
==> Updated Casks
Updated 11 casks.

Does anyone know if there's a flag to revert to old behavior and actually show the names of recently updated casks/formulas?

Thank you.

**Comment by jonchang** [comment, marked as answer]
You can still get this if you run `brew update` explicitly rather than having it run automatically as part of `brew upgrade`. This feature was disabled due to performance reasons.
